.github/workflows/test.yml

```
@@ -185,3 +185,25 @@ jobs:
       - name: Test NetworkX
         run: |
           pytest -n auto --doctest-modules --durations=10 --pyargs networkx
+
+  slow-tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v5
+      - name: Set up Python
+        uses: actions/setup-python@v6
+        with:
+          python-version: "3.13"
+
+      - name: Install packages
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install .[default,test]
+          python -m pip list
+
+      - name: Run slow tests
+        # NOTE: --runslow adds slow tests to the collection while -k slow
+        # filters out tests that are *not* marked slow, leaving only the
+        # slow tests
+        run: |
+          pytest -n auto --durations=20 --runslow -k slow --pyargs networkx
```

[Nit](https://docs.pytest.org/en/6.2.x/usage.html#specifying-tests-selecting-tests): `-k` matches keyword expressions; `-m` matches markers.
```suggestion
        # NOTE: --runslow adds slow tests to the collection while -m slow
        # filters out tests that are *not* marked slow, leaving only the
        # slow tests
        run: |
          pytest -n auto --durations=20 --runslow -m slow --pyargs networkx
```